src/credentials.ts

```diff
@@ -98,3 +105,111 @@ export class CredentialType {
     }
   }
 }
+
+export class CredentialKeeper
```

Comment:
Should the `CredentialKeeper` be responsible for the credential metadata as well. It's used in the `display` method, but we also need parts of `renderInfo` as well.

Reply:
I have updated the API to handle `CredentialMetadataSummary` objects instead, which have an issuer profile.
`CredentialType` now gives access to an `issuerProfile` and `renderAs` and `definition`